### ISSUE TITLE
TypeError: Cannot convert undefined or null to object for projects without dev dependencies in CLI

### ISSUE THREAD

**Issue by talentlessguy** [talentlessguy/tinyhttp#259]
**Describe the bug**

When a project doesn't have `devDependencies` field, it crashes like this:

```
Creating a new tinyhttp project from basic template in my-app folder ⚡
Fetching template contents ⌛
file:///home/v1rtl/n/pnpm-global/5/node_modules/.pnpm/@tinyhttp+cli@1.3.0/node_modules/@tinyhttp/cli/dist/index.js:63
    const depCount = (Object.keys(file.get('dependencies')) || []).length + (Object.keys(file.get('devDependencies')) || []).length;
                                                                                    ^

TypeError: Cannot convert undefined or null to object
    at Function.keys (<anonymous>)
    at installPackages (file:///home/v1rtl/n/pnpm-global/5/node_modules/.pnpm/@tinyhttp+cli@1.3.0/node_modules/@tinyhttp/cli/dist/index.js:63:85)
    at CAC.<anonymous> (file:///home/v1rtl/n/pnpm-global/5/node_modules/.pnpm/@tinyhttp+cli@1.3.0/node_modules/@tinyhttp/cli/dist/index.js:221:11)

```

**To Reproduce**

```
pnpm i -g @tinyhttp/cli

tinyhttp new basic my-app

cd my-app
```

**Versions**

* `node`: 16.2
* `@tinyhttp/app`: 1.3

**Additional context**

In order to fix we need to add `|| {}` inside of `Object.keys(...)`

**Comment by RhnSharma** [comment]
I would like to take this one.

**Comment by talentlessguy** [comment]
fixed in `@tinyhttp/cli@1.3.1`
